src/components/Breadcrumb/Breadcrumb.story.jsx

```diff
@@ -106,11 +106,15 @@ storiesOf('Watson IoT | Breadcrumb', module)
               <BreadcrumbItem href="#" title="3 A really long page name">
                 3 A really long page name
               </BreadcrumbItem>
-              <BreadcrumbItem href="#" title="4 Another page">
-                4 Another page
+              <BreadcrumbItem href="#" title={text('Breadcrumb 4 text', '4 Another page')}>
+                {text('Breadcrumb 4 text', '4 Another page')}
```

Comment:
I don't see this knob, only the knob at line 109. I'm guessing its because they have the same title.

Reply:
same reply as above.